### ISSUE TITLE
Shimmy

### ISSUE THREAD

**Issue by thewchan** [conda-forge/gymnasium-feedstock#8]
### Comment:

@h-vetinari Part of the challenge is also this: https://github.com/conda-forge/shimmy-feedstock/, which I have recently brought to conda forge in anticipation of Gymnasium 0.27. The thing is that there are many optional dependency there too, and I suppose we should support all those, but several of them are not on Conda yet. Including:

https://pypi.org/project/dm-control/
https://pypi.org/project/open-spiel/
https://pypi.org/project/PettingZoo/

At least, in my first search over in Conda Forge, I don't think those are in the catalogue yet. Thoughts?

**Comment by thewchan** [comment]
CC: @jkterry1
@pseudo-rnd-thoughts

**Comment by h-vetinari** [comment]
I'd like to first get the transition release of gym->gymnasium into conda-forge, i.e. 0.26.1 with all relevant extras. Then we can tackle shimmy and _that_ ecosystem...

**Comment by thewchan** [comment]
@h-vetinari good call, you mentioned gym last version and gymnasium 0.26.1 is essentially identical? Would the recipe require little changes then between the 2?

**Comment by h-vetinari** [comment]
> Would the recipe require little changes then between the 2?

That's the idea, yes. Which is why I want to "import" the history of the gym feedstock to have a more or less seamless transition from gym to gymnasium (and we can keep the old gym-outputs as wrappers for the gymnasium packages for a while, so people wanting to upgrade gym would get gymnasium automatically).

See discussions in https://github.com/conda-forge/gym-feedstock/pull/29 and https://github.com/conda-forge/staged-recipes/pull/21485

**Comment by thewchan** [comment]
@h-vetinari since you have access now (let me know of others to add) do you want to take the lead? Happy to help otherwise

**Comment by h-vetinari** [comment]
> @h-vetinari since you have access now (let me know of others to add) do you want to take the lead? Happy to help otherwise

Yes, I just need to get to a computer and have a moment. Give me a few hours.

**Comment by traversaro** [comment]
> https://pypi.org/project/dm-control/

Slightly related:
* https://github.com/conda-forge/staged-recipes/pull/21311
* https://github.com/conda-forge/staged-recipes/pull/21512

**Comment by pseudo-rnd-thoughts** [comment]
I must have missed this thread.
Are we able to just have the base install with non of the optional installs. 
Then we can iteratively add support for each of the modules

**Comment by pseudo-rnd-thoughts** [comment]
Shimmy is being removed as a non-necessary module in v0.27.1 but it would be nice to add support in the future.

**Comment by h-vetinari** [comment]
I opened an issue (plus a couple follow-ups) for the remaining shimmy optional dependencies: https://github.com/conda-forge/shimmy-feedstock/issues/7

@ChristofKaufmann, any chance of interesting you to help out with that? ;-)

**Comment by ChristofKaufmann** [comment]
@h-vetinari Maybe I could do PettingZoo... but I will be gone camping without laptop very soon. So, I am not sure yet, when I can do it.
